const errorLogFunc = () => {
  let testArray = ["name", "fatherName", "age", "education"];
  console.log(testArray[4])
};
console.log(errorLogFunc());
